sashimi-webapp/src/logic/filemanager/file.js

```diff
@@ -0,0 +1,133 @@
+import storage from '../../database/storage';
+import Folder from './folder';
+
+/**
+* File Object
+*
+* @param {Number} fileID
+* @param {String} fileName
+* @param {String} filePath
+*/
+
+
+export default function File(fileID, fileName, filePath, parentFolder) {
+  this.id = fileID;
+  this.name = fileName;
+  this.path = filePath;
```

Comment:
I'm not sure the intention of these parameters "this.something", but if these are private parameters it would be better to put them outside the "export". >.<